.env

```diff
@@ -116,13 +115,44 @@ MF_PROVISION_THINGS_LOCATION=http://things:8182
 MF_PROVISION_USER=
 MF_PROVISION_PASS=
 MF_PROVISION_API_KEY=
-MF_PROVISION_CERTS_SVC_URL=http://localhost/certs
+MF_PROVISION_CERTS_SVC_URL=
```

Comment:
Is this used?

Reply:
Yes

Reply:
Why you removed the ENVAR here?
https://github.com/mainflux/mainflux/pull/1188/files#diff-91be67728a6837cfd018981b01a662a0L47

Reply:
Can we let a default value?